params/protocol_params.go

```diff
@@ -184,8 +184,8 @@ const (
 	BlobTxTargetBlobGasPerBlock = 3 * BlobTxBlobGasPerBlob // Target consumable blob gas for data blobs per block (for 1559-like pricing)
 	MaxBlobGasPerBlock          = 6 * BlobTxBlobGasPerBlob // Maximum consumable blob gas for data blobs per block
 
-	BlobReserveThreshold      = 18 * (24 * 3600) / 3 // it keeps blob data available for 18 days in local.
-	BlobExtraReserveThreshold = 1 * (24 * 3600) / 3  // it adds more time for expired blobs for some request cases, like expiry blob when remote peer is syncing, default 1 day.
+	MinBlocksForBlobRequests           uint64 = 4096 * 32 * 12 / 3  // it keeps blob data available for ~18.2 days in local.
```

Comment:
Can you write in the comment what do 4096, 32 and 12 represent for easier understanding?

Reply:
ok, it ref the latest BEP changes.